.eslintrc.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
 			},
 		],
 		'@wordpress/no-unsafe-wp-apis': 'off',
+		'@wordpress/no-store-string-literals': 'warn',
```

Comment:
It'd be great to prefix the rule name with "data" so it better explains its intent:  
```suggestion
		'@wordpress/data-no-store-string-literals': 'warn',
```